src/App.js

```diff
@@ -1,21 +1,47 @@
 import TodoItem from "./components/TodoItem/TodoItem";
 import InputTodo from "./components/InputTodo";
+import FilterTodos from "./components/FilterTodos";
+
 import { useState, useEffect } from "react";
 
 function App() {
+  // useState:
   const [value, setValue] = useState("");
   const [todos, setTodos] = useState([]);
+  const [status, setStatus] = useState("all");
+  const [filteredTodos, setFilteredTodos] = useState([]);
 
+  // Fetch-Function:
   const getData = async () => {
     const response = await fetch("https://jsonplaceholder.typicode.com/todos");
     const data = await response.json();
     setTodos(data);
   };
 
+  // UseEffect:
   useEffect(() => {
     getData();
   }, []);
 
+  useEffect(() => {
+    const filterHandler = () => {
+      switch (status) {
+        case "completed":
+          setFilteredTodos(todos.filter((todo) => todo.completed === true));
```

Comment:
Instead define a variable and pass it to setFilteredTodos

for example

const variable = todos.filter((todo) => todo.completed === true)
setState(variable)